test/regress_dns.c

```diff
@@ -1183,17 +1186,27 @@ dns_initialize_nameservers_test(void *arg)
 		evdns_base_free(dns, 0);
 }
 #ifndef _WIN32
-#define RESOLV_FILE "empty-resolv.conf"
+#define RESOLV_FILE "./empty-resolv.conf"
```

Comment:
Consider using `regress_make_tmpfile` instead.

Reply:
Sure thing, I'll have to extend regress_make_tmpfile to return a filename on non-windows platforms.